src/mmw/apps/modeling/tasks.py

```diff
@@ -139,8 +146,33 @@ def format_catchment(srat_catchment):
                 'TotalP': srat_catchment['tploadrate_conc'],
                 'Sediment': srat_catchment['tssloadrate_conc'],
             },
+            'Loads': loads_template
         }
 
+    def add_load_by_key(load_type, value, load):
+        if load_type == 'tpload':
+            load['TotalP'] = value
+        elif load_type == 'tnload':
+            load['TotalN'] = value
+        elif load_type == 'tssload':
+            load['Sediment'] = value
+        else:
+            pass
+
+        return load
```

Comment:
Don't think we need to change this, but just to demonstrate, here's another way of doing the same thing that makes the mapping between their keys and ours clearer:

```python
def add_load_by_key(load_type, value, load):
    mapping = {
        'tpload': 'TotalP',
        'tnload': 'TotalN',
        'tssload': 'Sediment',
    }

    if load_type in mapping:
        load[mapping[load_type]] = value

    return load
```

Reply:
I like this mapping!